tests/smoke-go-group-multidir.yaml

```diff
@@ -182,10 +182,10 @@ output:
                   operation: update
                   support_file: false
                   type: file
-            pr-title: Bump the go_modules group across 2 directories with 1 update
+            pr-title: Bump the go_modules group group across 2 directories with 1 update
```

Comment:
I think this is a batch replace issue; this should not happen since this is the bug that @jurre was trying to fix

Reply:
The group name had "group" in it, so it came out as group group, I removed it!